go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -235,16 +204,12 @@ func newTxThrottler(env tabletenv.Env, config *txThrottlerConfig) (*TxThrottler,
 }
 
 // Open opens the transaction throttler. It must be called prior to 'Throttle'.
-func (t *TxThrottler) Open() error {
-	if !t.config.enabled {
-		return nil
-	}
-	if t.state != nil {
+func (t *TxThrottler) Open() (err error) {
+	if !t.config.enabled || t.state != nil {
```

Comment:
To be honest, I quite prefer the previous code. With this rewrite I find it more difficult to decipher the meaning of the early return. This is mostly due to having two negatives in this `if` statement.

Reply:
@shlomi-noach in that case I'm the odd one out 👍. Let me revert this change